README.md

```diff
@@ -75,51 +75,83 @@ Reworked into these six RFCs:
 ####Authentication
 coming shortly  
 
-####Versioning 
-(this needs further clarifications)
+####Versioning
 
 **Golden Rule** 
 
-		Once we've released an API, we must NOT make breaking changes to it
+    Once we've released an API, we must NOT make breaking changes to it.
 
-        (even if it's not being used, or if it's only deployed to staging, 
-        or if there's only one client, or if we marked it as beta, 
-        or if all of the clients are sitting in the same room as us, 
-        or if we're really-really-careful-yes-we- really-mean-it-this-time)
+The Golden Rule **MUST NOT** be broken apart from in *exceptional* circumstances, such as for legal reasons.
 
+The versioning stategy has changed, existing endpoints using the 'legacy' versioning will continue to be supported until deprecated.
 
-*Content Negotiation* 
+#####New Versioning Strategy
+
+All new endpoints **MUST** have a version number in their URL directly beneath the resource name.
+
+    e.g. /documents/v1/<id>
+
+Endpoints that include a version within the URL **MUST NOT** include a version within the media type.
+
+    application/vnd.mendeley-<RESOURCE_TYPE>+json
+    e.g. application/vnd.mendeley-document+json
+    
+Clients **MUST** send a non-null and non-wildcard Accept header for endpoints that produce content. Clients **MUST** also send a non-null Content-Type header for endpoints that consume content.
+
+####Legacy Versioning Strategy
 
 Use HTTP's content negotiation mechanism for versioning. Custom media types are used in the API to let the consumers choose the format of the data they wish to receive. This is done by adding the `Accept` header when you make a request. Media types are specific to resources, allowing them to change independently.  
```

Comment:
I'd change this to past tense, to emphasise the point more. "We used to use..." etc.